### PR TITLE
feat: leetcode Maximum Twin Sum of a Linked List solution

### PR DESCRIPTION
**Pull request by Sindhuinti**
- 

- [&#9745; ] #### Description of Change

#### Checklist

- [&#9745; ] Added description of change
- [ &#9745;] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [&#9745; ] Relevant documentation/comments is changed or added
- [ &#9745;] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [&#9745; ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ &#9745;] I acknowledge that all my contributions will be made under the project's license.

Notes: 

Add the leetcode question Maximum Twin Sum of a Linked List (2130) -medium level

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -4,4 +4,5 @@
     "editor.formatOnPaste": true,
     "editor.formatOnType": true,
     "files.insertFinalNewline": true,
+    "C_Cpp.errorSquiggles": "Disabled",
```
Comment (Panquesito7) [inline .vscode/settings.json:7]: Could you please revert the changes here? Thanks.
```suggestion
```

**Review by Sindhuinti — COMMENTED** [review]
Required changes made
